### DISCUSSION TITLE
Command-Line parameters for config/preferences?

### DISCUSSION THREAD

**Discussion by mojomojomojo** [VerySleepy/verysleepy#96] [Ideas]
Would it be wrong-headed to have command line parameters for items that are config items? The latter are saved (somehow) and will persist beyond a single invocation of Very Sleepy. However, if I wanted to do a lot with command-line options, I would need to configure the settings I wanted before I could do the scriptomatic stuff with sleepy.exe.

Would it be wrong to create command-line params for config items?

In particular, I'm thinking about the minidump options, but any of them might be something that I'd want to control via script.

**Comment by CyberShadow** [comment]
I don't see why not.

**Reply by mojomojomojo** [reply]
Would you expect that setting a config option on the command line would or would not get "saved" in the config?

I would not expect it to get saved, since it's kind of overriding whatever configured value there may be.

**Reply by CyberShadow** [reply]
Yep, same.
